docs/code/demos/advanced.py

```diff
@@ -40,18 +40,15 @@ def do_pipeline(arg1=Return('do_on_process'),
     Argument 'arg1' gets the return value of 'do_on_process'
     Argument 'arg2' gets the return value of function 'get_item'
     Argument 'arg3' is simply the value of a session parameter 'my_arg'"""
-    ...
 
 @app.task('daily & is foo', execution="thread")
 def do_custom():
     """This task runs once a day and when is_foo returns True
     This task runs on separate thread"""
-    ...
 
 @app.task('(true & true) | (false & True & ~True)')
 def do_complex():
     """Notice the logical expression in the task start condition"""
-    ...
```

Comment:
I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).